torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4105,6 +4107,11 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypesIfCUDA=all_types_and(torch.bool, torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_atan2,
            ),
+    OpInfo('atan2',
+           dtypes=_DYNAMIC_DTYPES,
```

Comment:
What's the mechanism where _DYNAMIC_DTYPES is replaced with the correct dtypes (and dtypesIfCUDA?) for atan2?

Reply:
I think for this PR, we don't want to do that. User can just get the dtypes with the tool and manual enter them and remove the `_DYNAMIC_DTYPES` while at it.

Like we have verified, the CI will fail if user forgets to remove `_DYNAMIC_DTYPES`.

Reply:
OK; see my offline suggestion for how we might be able to model this more simply